.gitlab/deploy_containers/conditions.yml

```diff
@@ -0,0 +1,85 @@
+
+# Rule to make job manual when running deploy pipelines, or automatic and on success on RC pipelines
+.manual_auto_on_rc:
```

Comment:
```suggestion
.manual_on_deploy_or_auto_on_rc:
```
It's maybe more explicit like this as you don't have the comment where the rule is used

Reply:
I can update it yes